screens/register.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { StyleSheet, Text, View, FlatList, Image, ScrollView, AppRegistry, Dimensions} from 'react-native';
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <FlatList
```

Comment:
This shouldn't be a `FlatList` because the user isn't supposed to scroll through a list of items. Instead, you should just hard code in separate `TextView` components.